treeherder/config/settings.py

```diff
@@ -22,7 +22,7 @@
 DEBUG = env.bool("TREEHERDER_DEBUG", default=False)
 
 # Papertrail logs WARNING messages. This env variable allows modifying the behaviour
-LOGGING_LEVEL = env.str("LOGGING_LEVEL", default='DEBUG' if DEBUG else 'WARNING')
+LOGGING_LEVEL = env.str("LOGGING_LEVEL", default='DEBUG' if DEBUG else 'INFO')
```

Comment:
@SuyashSalampuria Let's keep the PR to just this line. It might just be easier to start a new PR and throw this one away.

Actually just:
` LOGGING_LEVEL = env("LOGGING_LEVEL", default='INFO')`

Reply:
Although I have done what you asked for
I did not understand why it would be easier to start a new PR and what must be done in that

Reply:
I wanted to save you some work from having to modify a lot of files back to their original state.